tests/conftest.py

```diff
@@ -117,12 +118,65 @@ def make_data_file(
             run=run,
         )
 
-    df_6_seasonal = make_data_file(
-        unique_id="tasmin_sClim_BNU-ESM_historical_r1i1p1_19650101-19701230", run=run3,
+    df_bnu_seasonal = make_data_file(
```

Comment:
This is great that you've set up all of this test data in the database!

The fixture, however, still reads with a lot of repetition and redundancy. Is there anyway you can tighten it up? Options:

1. Are the unique_id's used in the tests? If not, maybe just generate a random hash to save space
2. Most of the filenames share the same base path. Factor that out and put the replace the base path in an fstring
3. Maybe at the end, instead of enumerating all of the data files, you could do it programatically, by something like this:

```python3
data_files = [ v for k, v in locals().items() if k.startswith('df') ]
```
I'm sure that there may be other ways to tighten it up. Good use of the `make_data_file()` function though.